src/components/Node/Node.wrapper.tsx

```diff
@@ -32,16 +32,16 @@ import { INodeDefaultProps, NodeDefault } from './Node.default'
 export interface INodeWrapperProps {
   config: IConfig
   node: INode
-  Component: React.FunctionComponent<INodeDefaultProps>
+  Component?: React.FunctionComponent<INodeDefaultProps>
   offset: IPosition
   selected: ISelectedOrHovered | undefined
   hovered: ISelectedOrHovered | undefined
   selectedLink: ILink | undefined
   hoveredLink: ILink | undefined
   isSelected: boolean
-  NodeInner: React.FunctionComponent<INodeInnerDefaultProps>
-  Ports: React.FunctionComponent<IPortsDefaultProps>
-  Port: React.FunctionComponent<IPortDefaultProps>
+  NodeInner?: React.FunctionComponent<INodeInnerDefaultProps>
+  Ports?: React.FunctionComponent<IPortsDefaultProps>
+  Port?: React.FunctionComponent<IPortDefaultProps>
```

Comment:
Ze zijn toch juist verplicht?

```suggestion
  NodeInner: React.FunctionComponent<INodeInnerDefaultProps>
  Ports: React.FunctionComponent<IPortsDefaultProps>
  Port: React.FunctionComponent<IPortDefaultProps>
```

Misschien even overleggen, ik zie dat je het op andere plekken ook hebt aangepast.